### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tanishq-xin**
Changes grammatically,
Formatted markdown (+ divisions)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,78 @@
...
 ## Features
 
--   Fast
--   Object Oriented
--   Typescript, ESM, CJS support
--   Easy to Use
+- **Fast Performance**
+- **Object-Oriented Design**
+- **Support for TypeScript, ESM, and CJS**
+- **User-Friendly Interface**
```
Comment (FC5570) [inline README.md:16]: Friendly UI doesn't sound suitable for a backend lib imo.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,78 @@
...
 });
 ```
 
 ## Links
 
--   Documentation (Coming Soon)
--   [Discord Server](https://discord.gg/qchtmGDdFr)
--   [Github Organization](https://github.com/spotifyts)
--   [NPM Page](https://npmjs.com/spotify.ts)
+- [Documentation (Coming Soon)](https://github.com/spotifyts/spotify.ts)
```
Comment (FC5570) [inline README.md:72]: This is meant for the docusaurus documentation site. User's can find their way through the github org link.

**Review by FC5570 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tanishq-xin) [reply README.md:72]: so shall I remove this it then?

Comment (FC5570) [reply README.md:72]: yes

Comment (tanishq-xin) [reply README.md:72]: to this right?
image

**Pull request merged**